doc/api.yml

```diff
@@ -1181,6 +1257,111 @@ paths:
           $ref: '#/components/responses/429'
         '500':
           $ref: '#/components/responses/500'
+  '/students/{student_id}/pointing':
+    post:
+      tags:
+        - Check-in
+      summary: Create a pointing of one student
+      operationId: createPointing
+      parameters:
+        - name: student_id
+          in: path
+          required: true
+          description: give the student id
+          schema:
+            type: string
+      requestBody:
+        description: Pointing student to create
```

Comment:
Rename.

Reply:
change to `Attendance movement to create`